### ISSUE TITLE
Track Ammo In Database

### ISSUE THREAD

**Issue by SGanthony** [EpochModTeam/DayZ-Epoch#1018]
DayZ Epoch really needs to track the amount of ammo in each magazine via the database to prevent further exploitation and add more value to the in game money system. Right now it is way too easy for a person to just relog with tons of partially spent ammo and then come back with them filled, thus totally circumventing the expense. To complement this, players should be able to combine ammo.

If I even remotely knew how to code this I would. I would be willing to help with this in any way my newbie coding can help. I can copy/paste and do repetitive data entry pretty well!

**Comment by icomrade** [comment]
It's not possible to pull the ammo count out of the player's backpack without opening the players gear ui then opening the backpack (this process is visible and takes less than a second but is annoying). This would have to be done every time the player is saved, which is very often. The other problem is that this process is extremely unreliable when the player is logging out, causing them to either spawn in with nothing or the incorrect amount of items.

In short, closed because it's implausible in ArmA 2/OA

**Comment by ebaydayz** [comment]
Foxy said he is working on an extension to do this in vanilla. If that ever gets released we can implement it here too.
